docs/additional-configuration.adoc

```diff
@@ -215,6 +215,16 @@ kubectl create secret -n "$NAMESPACE" generic git-ssh-key \
   --from-file=ssh_config=/tmp/ssh_config \
   --from-literal=passphrase="$PASSPHRASE"
 ----
++
+Note: If a passphrase is provided, the DevWorkspace operator adds a postStart event that starts the SSH agent and adds the passphrase.
```

Comment:
For consistency with the rest of the document: "DevWorkspace operator" -> "DevWorkspace Operator"

Reply:
Also, maybe we should bold "Note:"?
i.e. `Note:` -> `*Note:*`

Reply:
Fixed, I've also updated all `Note:` do be bold